src/css/components/icon.scss

```diff
@@ -0,0 +1,15 @@
+
+.icon-container {
+  height: 3.125rem;
+  width: 3.9rem;
+}
+
+.icon {
+  max-height: 3.1rem;
+}
+
+.icon-alt {
+  .cls-1 {
```

Comment:
I see this is everywhere in the .svg. Did they automatically give you that name? If not, what about `.cls-stroke`?